utils/build/docker/python/flask/debugger_controller.py

```diff
@@ -0,0 +1,128 @@
+from flask import Blueprint, request, abort
+from debugger.pii import Pii, CustomPii
+from debugger.expression_test_struct import ExpressionTestStruct
+from debugger.collection_factory import CollectionFactory
+
+# The `debugger` feature allows attachment to specific lines of code.
+# Due to differences in line numbering between libraries,
+# 'dummy lines' are used to standardize this functionality.
+# dummy line
+# dummy line
+# dummy line
+debugger_blueprint = Blueprint("debugger", __name__, url_prefix="/debugger")
+
+intLocal = 0
+intMixLocal = 0
+
+
+@debugger_blueprint.route("/log", methods=["GET"])
```

Comment:
could you document in `docs/weblog` what those route are supposed to perform ?

Reply:
It's actually not the new endpoint. It was not implemented for `python` but already exist for `dotnet` and`java`.
All the documentation already there.
https://github.com/DataDog/system-tests/tree/main/docs/weblog#get-debugger